exporter/awsemfexporter/metric_translator.go

```diff
@@ -362,12 +370,115 @@ func updateGroupedMetric (dp DataPoint, pMetricData *pdata.Metric, key string, g
 				zap.String("DataType", pMetricData.DataType().String()),
 				zap.String("Name", pMetricData.Name()),
 				zap.String("Unit", pMetricData.Unit()),)
-		groupedMetricMap[key].Metrics[metricName] = &metricInfo
 	} else {
 		groupedMetricMap[key].Metrics[metricName] = &metricInfo
 	}
 }
 
+// rate is calculated by valDelta / timeDelta
```

Comment:
could we move this logic back down below so that I can see the diff changes directly with the original code?

Reply:
resolved